packages/babel-preset-env/data/built-in-modules.json

```diff
@@ -3,10 +3,10 @@
     "edge": "16",
     "firefox": "60",
     "chrome": "61",
-    "safari": "10.1",
+    "safari": "11",
     "opera": "48",
-    "ios_saf": "10.3",
-    "and_chr": "71",
-    "and_ff": "64"
+    "ios_saf": "11.0",
+    "and_chr": "74",
```

Comment:
I'm unsure why these `and` entries changed when I ran the script - I guess the data for `caniuse` has shifted a bit?

Reply:
reverted `and` changes